src/libcore/hash/sip.rs

```diff
@@ -19,15 +19,17 @@ use super::Hasher;
 ///
 /// See: http://131002.net/siphash/
 ///
-/// Consider this as a main "general-purpose" hash for all hashtables: it
-/// runs at good speed (competitive with spooky and city) and permits
-/// strong _keyed_ hashing. Key your hashtables from a strong RNG,
-/// such as `rand::Rng`.
+/// This is the default hashing function used by standard library (eg.
```

Comment:
Perhaps this could say "this is *currently* the default" to emphasize that it may change over time?